### PR TITLE
DN-04: Setting up the project structure with basic package installation

### PR DESCRIPTION
**Pull request by puncoz**
**Issue:** https://github.com/IdeaGhar/DataNepal/issues/4

**Reasons to do this task :**
* Project Scaffolding and setting up the project

**Main changes are :**
- [x] barryvdh/laravel-ide-helper added for phpstorm support 
- [x] graham-campbell/exceptions for informative laravel error pages
- [x] rap2hpoutre/laravel-log-viewer added to view error logs via /logs routes
- [x] spatie/laravel-activitylog for logging activities.
- [x] Resolved Laravel migration: unique key is too long error
- [x] Auth Scaffolding
- [x] Folder Structure
- [x] zizaco/entrust for Role based Authentication

**Task Status** 
- [x] Work In Progress
- [x] Completed

**Code Review**
**Assignee:** @sadhakbj

### CODE REVIEW

**Comment by sadhakbj** [comment]
Please Update the Branch Name to DN:04 only.  Details to be provided in the link.

File: app/Providers/AppServiceProvider.php [hunk 1]
```diff
@@ -10,10 +11,10 @@ class AppServiceProvider extends ServiceProvider
      * Bootstrap any application services.
      *
      * @return void
-     */
+     */a
```
Comment (sadhakbj) [inline app/Providers/AppServiceProvider.php]: Remove this `a`

File: config/ide-helper.php [hunk 2]
```diff
@@ -0,0 +1,168 @@
+<?php
+
+return array(
```
Comment (sadhakbj) [inline config/ide-helper.php]: I think using [  ] for representing array is better than array( ). Laravel by default uses [ ] , which insures the consistency in our code.

File: config/ide-helper.php [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    |--------------------------------------------------------------------------
+    |
+    | Include helper files. By default not included, but can be toggled with the
+    | -- helpers (-H) option. Extra helper files can be included.
+    |
+    */
+
+    'include_helpers' => false,
+
+    'helper_files' => array(
```
Comment (sadhakbj) [inline config/ide-helper.php]: Same here.

File: app/Providers/AppServiceProvider.php (hunk 1 above)
Comment (puncoz) [inline app/Providers/AppServiceProvider.php]: Blunder happened :)

**Pull request merged**